x-pack/plugins/security_solution/public/common/mock/endpoint/app_context_render.tsx

```diff
@@ -115,7 +124,7 @@ export const createAppRootMockRenderer = (): AppContextTestRender => {
   const AppWrapper: React.FC<{ children: React.ReactElement }> = ({ children }) => (
     <KibanaContextProvider services={startServices}>
       <AppRootProvider store={store} history={history} coreStart={coreStart} depsStart={depsStart}>
-        {children}
+        <QueryClientProvider client={queryClient}>{children}</QueryClientProvider>
```

Comment:
Maybe not for this PR, but I would actually suggest moving this up to the app level. some of our components are/will be consumed from other areas of security solution, so moving it up would set us up for it. It would also allow others on the team to use it if they wished